test/e2e/tests/account/unlock-wallet.spec.ts

```diff
@@ -16,44 +16,6 @@ import PrivacySettings from '../../page-objects/pages/settings/privacy-settings'
 import HeaderNavbar from '../../page-objects/pages/header-navbar';
 import ChangePasswordPage from '../../page-objects/pages/settings/change-password-page';
 
-async function doPasswordChangeAndLockWallet(
-  driver: Driver,
-  currentPassword: string,
-  newPassword: string,
-  isSocialLogin: boolean = false,
-) {
-  // navigate to security & privacy settings
-  const headerNavbar = new HeaderNavbar(driver);
-  await headerNavbar.openSettingsPage();
-  const settingsPage = new SettingsPage(driver);
-  await settingsPage.checkPageIsLoaded();
-  await settingsPage.goToPrivacySettings();
-
-  const privacySettings = new PrivacySettings(driver);
-  await privacySettings.checkPageIsLoaded();
-  await privacySettings.openChangePassword();
-
-  const changePasswordPage = new ChangePasswordPage(driver);
-  await changePasswordPage.checkPageIsLoaded();
-
-  await changePasswordPage.confirmCurrentPassword(currentPassword);
-
-  await changePasswordPage.changePassword(newPassword);
-  if (isSocialLogin) {
-    await changePasswordPage.checkPasswordChangedWarning();
-    await changePasswordPage.confirmChangePasswordWarning();
-  }
-
-  await privacySettings.checkPasswordChangeSuccessToastIsDisplayed();
-
-  await settingsPage.closeSettingsPage();
-
-  // Wait for the password change to be applied to the social login user
-  await driver.delay(2_000);
-
-  await headerNavbar.lockMetaMask();
-}
-
```

Comment:
ℹ️  this was just used in one place, so there's no need to have a function fo r that. I've just moved it into the spec below for better clarity